src/components/button/examples/button-disabled-vs-hidden.tsx

```diff
@@ -0,0 +1,160 @@
+                not currently available but may become available in the future.
+                For example, you can disable a "Submit" button until all
+                required form fields are filled out.
+            </p>,
+            <div class="do">
+                <limel-header icon="ok" heading="Disabled button in the form" />
+                <limel-checkbox
+                    label="Accept terms and conditions"
+                    required
+                    onChange={this.setChecked}
+                    checked={this.value}
+                />
+                <limel-button
+                    label="Submit"
+                    disabled={this.value ? false : true}
+                />
+            </div>,
+            <p>
+                It's helpful when you want to communicate to the user why they
+                can't perform a certain action. Disabled buttons could have a
+                tooltip or message explaining why they are disabled.
+            </p>,
+
+            <div class="do">
+                <limel-header
+                    icon="ok"
+                    heading="Disabled button together with a Tooltip"
+                />
+                <div id="tooltip-example">
+                    <limel-button
+                        class="disabled-button"
+                        disabled
+                        icon="phone"
+                        label="Call"
+                    />
+                </div>
+                <limel-tooltip
+                    label="Select recipient to make a call"
+                    elementId="tooltip-example"
+                />
+            </div>,
+            <p>
+                It can be suitable for indicating to the user that they need to
+                complete a certain step or meet specific conditions before
+                proceeding.
+            </p>,
+            <div class="do">
+                <limel-header
+                    icon="ok"
+                    heading="Disabled button together with an info icon"
+                />
+                <div class="button-icon">
+                    <limel-button
+                        class="disabled-button"
+                        disabled
+                        icon="plus_math"
+                        label="Add recipient"
+                    />
+                    <limel-icon name="info" id="tooltip" size="x-small" />
+                </div>
+                <limel-tooltip
+                    label="To enable the feature call our support"
+                    elementId="tooltip"
+                />
+            </div>,
+            <h3>Hiding a Button:</h3>,
+            <p>
+                Hide the button if it's not relevant to a context. If the user
+                is already assigned there is no sense in having the assign
+                button, it is better to show the <b> unassign</b> button
+                instead.
```

Comment:
```suggestion
                Sometimes, displaying a disabled element does not make sense.
                This could be of course due to various reasons, 
                but a common scenario is coexistence of another "enabled" element 
                that does the opposite of what the disabled element does.
                
                In this example, there is no point to show two buttons, one for 
                "assigning" and one for "un-assigning". 
                The "Assign" button should be hidden if it's not relevant to a context. 
                If the user is already assigned there is no sense in having the assign
                button, it is better to show the <b>Unassign</b> button
                instead.
```